CONTRIBUTING.md

```diff
@@ -87,77 +87,145 @@ You have done! Now it is possible to insert breakpoints and debug server and cli
 
 ## Setup additional components in development environment
 
-### Automatic annotation
--   Install OpenVINO on your host machine according to instructions from
-[OpenVINO website](https://docs.openvinotoolkit.org/latest/index.html)
+### DL models as serverless functions
 
--   Add some environment variables (copy code below to the end of ``.env/bin/activate`` file):
-```sh
-    source /opt/intel/openvino/bin/setupvars.sh
+Install [nuclio platform](https://github.com/nuclio/nuclio)
 
-    export OPENVINO_TOOLKIT="yes"
-    export IE_PLUGINS_PATH="/opt/intel/openvino/deployment_tools/inference_engine/lib/intel64"
-    export OpenCV_DIR="/usr/local/lib/cmake/opencv4"
-    export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/intel/openvino/inference_engine/lib/intel64"
+- You have to install `nuctl` command line tool to build and deploy serverless
+functions. Download [the latest release](https://github.com/nuclio/nuclio/releases).
+- The simplest way to explore Nuclio is to run its graphical user interface (GUI)
+of the Nuclio dashboard. All you need in order to run the dashboard is Docker. See
+[nuclio documentation](https://github.com/nuclio/nuclio/blob/development/README.md)
+for more details.
+
+```bash
+docker run -p 8070:8070 -v /var/run/docker.sock:/var/run/docker.sock \
+    -v /tmp:/tmp --name nuclio-dashboard quay.io/nuclio/dashboard:stable-amd64
```

Comment:
The command works once and run in background.
Next tries to run it I get:
```
docker: Error response from daemon: Conflict. The container name "/nuclio-dashboard" is already in use by container "8079afd744788faa2f04c195e3333cbab8a28cb7cdb0d417c86f7b7748829704". You have to remove (or rename) that container to be able to reuse that name.
```

Again, not everyone understands how docker works and that for example need to remove the container.

Reply:
Also i guess it could be useful to add `--restart=always` flag to start nuclio automatically